code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -610,32 +768,62 @@
 	held_index = 2
 	px_x = 6
 	px_y = 0
+	can_be_disabled = TRUE
 
-/obj/item/bodypart/r_arm/is_disabled()
-	if(HAS_TRAIT(owner, TRAIT_PARALYSIS_R_ARM))
-		return BODYPART_DISABLED_PARALYSIS
-	return ..()
+/obj/item/bodypart/r_arm/set_owner(new_owner)
+	. = ..()
+	if(. == FALSE)
+		return
+	if(owner)
+		if(HAS_TRAIT(owner, TRAIT_PARALYSIS_R_ARM))
+			ADD_TRAIT(src, TRAIT_PARALYSIS, TRAIT_PARALYSIS_R_ARM)
+			RegisterSignal(owner, SIGNAL_REMOVETRAIT(TRAIT_PARALYSIS_R_ARM), PROC_REF(on_owner_paralysis_loss))
```

Comment:
this behavior seems kinda copypasta, can you make it into a var on the bodypart type for like "unique paralysis trait"